### PR TITLE
merge release/live service/61.1 into master

### PR DESCRIPTION
**Pull request by JessiePotter1**

### CODE REVIEW

File: DFC.App.JobProfile.AVService/AVAPIService.cs
Comment (JamesPrior-07) [inline DFC.App.JobProfile.AVService/AVAPIService.cs:1]: Add XML Comments

File: DFC.App.JobProfile.AVService/ApprenticeshipVacancyApi.cs
Comment (JamesPrior-07) [inline DFC.App.JobProfile.AVService/ApprenticeshipVacancyApi.cs:1]: Add XML Comments

File: DFC.App.JobProfile.MessageFunctionApp/Services/RefreshService.cs
Comment (JamesPrior-07) [inline DFC.App.JobProfile.MessageFunctionApp/Services/RefreshService.cs:1]: Add XML Comments

File: DFC.App.JobProfile.ProfileService/JobProfileService.cs
Comment (JamesPrior-07) [inline DFC.App.JobProfile.ProfileService/JobProfileService.cs:1]: Add XML Comment to GetApprenticeship Method

**Review by JamesPrior-07 — APPROVED** [review]
Reviewed and Approved

**Review by thomas-petty-dfe — APPROVED** [review]
Approved as these have been reviewed previously

**Pull request merged**